### DISCUSSION TITLE
Project Excluded Error Fix is in Progress | Our Teams are working on it.

### DISCUSSION THREAD

**Discussion by tuminzee** [fineanmol/Hacktoberfest2021#838] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/58180803/135720331-213cfd0c-7c4b-404f-ab11-7eddd7f9d283.png)

**Comment by ShreyaChopra13** [comment]
I am getting the same message. Can anyone tell what this means?

**Comment by fineanmol** [comment]
Hello All, 
No worries, We are working with the concerned team, This issue is in progress and will be resolved Shortly. ✨
Thank you for being patient.

**Comment by tuminzee** [comment]
> Hello All, No worries, We are working with the concerned team, This issue is in progress and will be resolved Shortly. ✨ Thank you for being patient.

Thank you for your response.

**Comment by prabaljainn** [comment]
Thanks for your Response

**Comment by manisha069** [comment]
Hello, I am maintainer as well, had listed your repo in my README in a curated list of repos to contribute to..

how are you working towards fixing the "excluded project" issue? Any insight would be valuable.

**Comment by SamikshyaMstar** [comment]
@fineanmol, still showing an excluded project, I guess u should add the Hacktober fest -accepted label, maybe that can help.

**Comment by manisha069** [comment]
@SamikshyaMstar unfortunately that wouldn't help either.. when a repo is excluded all PRs having hacktoberfest-accepted label and merged as well do not count.

**Comment by fineanmol** [comment, marked as answer]
> SamikshyaMstar unfortunately that wouldn't help either.. when a repo is excluded all PRs having hacktoberfest-accepted label and merged as well do not count.

My Team has contacted the support of digital ocean, and according to them, those repositories which are a fast way to create pull requests to complete hacktoberfest are not going to be considered in the FEST. 

So here is what we are gonna do, In the 3rd week of October, we are going to change the code of this repo to some of our previous projects. And after that, every pull request made to this repo will be auto included in the event. 🤩✨
